crml/staking/src/lib.rs

```diff
@@ -1473,6 +1467,24 @@ impl<T: Trait> Module<T> {
 		maybe_new_validators
 	}
 
+	/// Compute current `total_payout` and `max_payout` for specific era duration
+	pub fn current_total_payout(total_tokens: RewardBalanceOf<T>) -> (RewardBalanceOf<T>, RewardBalanceOf<T>) {
```

Comment:
for consistency: `total_tokens` -> `total_issuance`

Can also accept validators as a parameter here like with the `split_evenly_to_all_...` function

Reply:
The main reason for pulling the code out into `fn current_total_payout` was to reuse this function for our integration tests so we wanted to have minimal parameters to be passed (and rely on getter methods inside the function to have whatever we need in scope)

Reply:
Yeah, also I think the validators should same with `current_elected`

Reply:
Will be addressed in a follow up PR